tunnel/intra/direct_split.go

```diff
@@ -6,51 +6,38 @@ import (
 )
 
 type splitter struct {
-	DuplexConn
-	conn *net.TCPConn
+	*net.TCPConn
 	used bool // Initially false.  Becomes true after the first write.
 }
 
 // DialWithSplit returns a TCP connection that always splits the initial upstream segment.
 // Like net.Conn, it is intended for two-threaded use, with one thread calling
 // Read and CloseRead, and another calling Write, ReadFrom, and CloseWrite.
```

Comment:
Does this comment need to be updated per the interface change?

Reply:
I think this comment is still accurate.